game_eggs/steamcmd_servers/space_engineers/torch/egg-space-engineers--torch-server.json

```diff
@@ -0,0 +1,184 @@
+            "container": "debian:bullseye-slim",
```

Comment:
This is a steam game is there any reason this is not using the debian installer image so the apt install and update are not needed?

Reply:
Oh, i missed that. The image was neede in past for Wine install